translations/ui-dashboard/en_US.json

```diff
@@ -115,5 +115,7 @@
     "simpleSearch.widget.linkTextForWidget": "{linkText} for widget: {widgetName}",
     "dashboard.deleteWidget": "Delete widget",
     "dashboard.delete": "Delete",
-    "dashboard.deleteWidgetConfirmMessage": "Widget <strong>{name}</strong> will be <strong>deleted</strong>."
+    "dashboard.deleteWidgetConfirmMessage": "Widget <strong>{name}</strong> will be <strong>deleted</strong>.",
+    "simpleSearch.widget.noResultFound": "No results found"
```

Comment:
When adding a new translation we don't update the en_US file, only when changing an existing one. Can you remove this change please?

Reply:
sure